src/AddIns/Uno.UI.MediaPlayer.Skia.Gtk/GTKMediaPlayer.cs

```diff
@@ -212,79 +216,34 @@ private void UpdateVideoStretch()
 			}
 		});
 	}
+	protected void UpdateTransportControlHeight()
+	{
+		UIElement? container = VisualTreeHelper.GetParent(_videoContainer) as UIElement;
```

Comment:
I don't think this is a great way to find the transport control height. I think this code should probably go into one of the other classes that gets a reference to the transport controls and then sets a property on the GTKMediaPlayer with the height of the transport controls.

Reply:
you fixed creating the SetTransportControlsBounds